test/init.js

```diff
@@ -15,4 +15,8 @@ describe('init', function() {
         assert.throws(function() { R.init(undefined); }, TypeError);
         assert.throws(function() { R.init(); }, TypeError);
     });
+    it('handles array-like object', function() {
+        var args = (function() { return arguments; }(1, 2, 3, 4, 5));
+        assert.deepEqual(R.init(args), [1, 2, 3, 4]);
+    });
```

Comment:
important to note that unless the array-like object has its own `slice` method, `R.slice` *always* returns an Array, not a "sliced" version of the input array-like object.